recipes/ios-cmake/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+
+from conans import ConanFile, tools
+
+import os
+import platform
+import copy
+
+
+class IosCMakeConan(ConanFile):
+    name = "ios-cmake"
+    license = "BSD-3-Clause"
+    settings = "os" , "arch"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/leetal/ios-cmake"
+    options = {
+        "enable_bitcode": [True, False],
+        "enable_arc": [True, False],
+        "enable_visibility": [True, False],
+        "enable_strict_try_compile": [True, False],
+        "ios_target": ["OS", "OS64", "OS64COMBINED",
+                       "SIMULATOR", "SIMULATOR64",
+                       "TVOS", "TVOSCOMBINED",
+                       "SIMULATOR_TVOS", "WATCHOS",
+                       "WATCHOSCOMBINED", "SIMULATOR_WATCHOS"]
+    }
+    default_options = {
+        "enable_bitcode": False,
+        "enable_arc": False,
+        "enable_visibility": False,
+        "enable_strict_try_compile": False,
+        "ios_target": "OS64COMBINED", 
+    }
+    description = "ios Cmake toolchain to (cross) compile macOS/iOS/watchOS/tvOS"
+    url = "https://github.com/leetal/ios-cmake"
+    exports_sources =  "cmake-wrapper"
+
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    @staticmethod
+    def _chmod_plus_x(filename):
+        if os.name == 'posix':
+            os.chmod(filename, os.stat(filename).st_mode | 0o111)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("ios-cmake-{}".format(self.version), self._source_subfolder())
+
+    def build(self):
+        pass # there is nothign to build
+
```

Comment:
```suggestion
```

Reply:
I like this as documentation reasons, for readers, saying, there is nothing to do 
is there an alternative instead of using pass?

Reply:
`pass` is the python way to do it. So your code was correct.
(If you really don't want to remove the function: `nothign` -> `nothing`)

Reply:
Conan actually emits a message like "no build() step" when it's omitted which provides for feed back for users than the pass approach

Reply:
yes, I know, but I do not want to have this message since it make it look like a problem, and have it also as documentation
what I did not understand is, `nothing -> nothing` @madebr  mentioned

Reply:
There is a spelling error in the comment